### DISCUSSION TITLE
Use WebFrontEnd from Ubuntu (bare-metal or wsl2)

### DISCUSSION THREAD

**Discussion by RanchoHam** [udo-munk/z80pack#428] [General]
I have the z80pack installed on both bare-metal(HP800) and WSL2(MS Surface Pro) machines.  How does one use the WebFrontEnd?  (Sorry for the nubie question.)

**Comment by udo-munk** [comment]
If you are using versions 1.38 or 1.39 then the web server is builtin in IMSAI 8080 and Cromenco Z-1, but not enabled by default. To enable execute the machines with option -n, also try option -h, which shows help for all options.
If the machines are started with -n they will print the URL of the web server under the banner:

Web server builtin, URL is http://localhost:8080

So start a web browser on the system and point it to this URL. To connect from some other system you need to find out the IP address of the one running the machine emulation. On Linux systems the command ifconfig will tell you the IP addresses for the interfaces. Won't work if Linux is running under WSL2, the IP interfaces can not be accessed from the network.

**Reply by RanchoHam** [reply]
Thanks @udo-munk (& sorry for the double post)
